components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -743,8 +694,8 @@ std::string BraveReferralsService::FormatExtraHeaders(
     return std::string();
 
   const base::DictionaryValue* request_headers_dict = nullptr;
-  if (!GetMatchingReferralHeaders(*referral_headers_list, &request_headers_dict,
-                                  url))
+  if (!BraveReferralsHeaders::GetInstance()->GetMatchingReferralHeaders(
```

Comment:
oh, this is interesting. Do we serve any headers from `/promo/initialize/nonua"`? @fmarier

Reply:
good catch. no we don't (i authored the current version of laptop-updates). going to remove this too

Reply:
fixed

Reply:
confirmed with @aekeus this was good to remove